README.md

```diff
@@ -309,7 +310,7 @@ The `cri` plugin has reached GA status, representing that it is:
 * Passes all [node e2e tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-node/e2e-node-tests.md).
 * Passes all [e2e tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/e2e-tests.md).
 
-See results on the containerd k8s [test dashboard](https://k8s-testgrid.appspot.com/sig-node-containerd)
+See results on the containerd k8s [test dashboard](https://prow.k8s.io/?repo=containerd%2Fcontainerd)
```

Comment:
```suggestion
See results on the containerd k8s [test dashboard](https://testgrid.k8s.io/sig-node-containerd)
```
Is the new link for the broken one, if we want to refer to all sig-node-containerd tests

we can use `https://testgrid.k8s.io/containerd`,  but its already referred in https://github.com/containerd/containerd#kubernetes-k8s-ci-dashboard-group.

Reply:
+1, it'd be good to still use the testgrid here.

Reply:
Updated to use https://testgrid.k8s.io/containerd